### PR TITLE
JSM: Revert to *.prototype.is* = true

### PR DESCRIPTION
**Pull request by linbingquan**
Related issue: https://github.com/mrdoob/three.js/pull/21293

### CODE REVIEW

File: examples/jsm/renderers/webgpu/WebGPUSampledTexture.js [hunk 1]
```diff
@@ -59,10 +58,10 @@ class WebGPUSampledCubeTexture extends WebGPUSampledTexture {
 
 		this.dimension = GPUTextureViewDimension.Cube;
 
-		Object.defineProperty( this, 'isSampledCubeTexture', { value: true } );
-
 	}
 
 }
 
+WebGPUSampledCubeTexture.prototypisSampledCubeTexturee = true;
```
Comment (Mugen87) [inline examples/jsm/renderers/webgpu/WebGPUSampledTexture.js:65]: This is correct.

File: examples/jsm/renderers/webgpu/WebGPUUniform.js [hunk 2]
```diff
@@ -37,12 +37,12 @@ class FloatUniform extends WebGPUUniform {
 		this.boundary = 4;
 		this.itemSize = 1;
 
-		Object.defineProperty( this, 'isFloatUniform', { value: true } );
-
 	}
 
 }
 
+FloatUniform.prototype.isVector4Uniform = true;
```
Comment (Mugen87) [inline examples/jsm/renderers/webgpu/WebGPUUniform.js:44]: The flags in this file are completely wrong...

Comment (Mugen87) [reply examples/jsm/renderers/webgpu/WebGPUUniform.js:44]: Friendly reminder: https://github.com/mrdoob/three.js/pull/20100#issuecomment-780479068

**Review by linbingquan — COMMENTED** [review]
This my problem again, I didn't reviewed my code.
I had check it this time. @Mugen87 Thank you for your friendly reminder.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**